### DISCUSSION TITLE
Nvidia docker image through HA addon?

### DISCUSSION THREAD

**Discussion by ideal2545** [blakeblackshear/frigate#1953] [Ask A Question]
Hey all, 

Just curious if there was a way to use the nvidia docker image through an HA addon or if I need to go a more manual route with portainer and all that with my bare metal HASSOS install.

**Comment by wes1993** [comment]
Hello,
have you find a solution for this?

Best regards
Stefano

**Reply by blakeblackshear** [reply]
Beta version includes nvidia support without needing a separate image, so it should just work.

**Reply by wes1993** [reply]
Thanks for your reply :)
Do you know if also support old GPU? 
I can use only CUDA 9.1 newer are not supported with my GPU (GeForce 820M)
